hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OzoneAclUtil.java

```diff
@@ -47,24 +49,44 @@ public final class OzoneAclUtil {
   private OzoneAclUtil() {
   }
 
+  private static ACLType[] userRights;
```

Comment:
this static initialization need to be protected, else can be initalized multiple time in multi-threaded env. There is no much impact as its re-init with same value only.
If not planned protect, order of groupRights init and userRights init should be controlled, else one of member can be null for other threads as per if-check.

Reply:
Yeah, from performance point, it's not planned to protect the initialization of userRights and groupRights.  The consequences is userRights and groupRights may be initialized several times, which is fine.